.gitignore

```diff
@@ -1 +1,5 @@
 target/
+
+
+# jetbrains
+.idea
```

Comment:
Quick comment on the draft: can you put this bit into a separate PR?

Reply:
I understand wanting to keep PRs succinct and indexable, but I think this is fine in this instance. It's not related to our own code at all and feels excessive to spend extra time breaking it out into another PR.